.ci/matrix_job.yaml

```diff
@@ -102,6 +103,33 @@ steps:
       echo
     parallel: false
 
+  - name: Install Doca-host
+    containerSelector:
+      - "{category: 'base'}"
+    agentSelector:
+      - "{nodeLabel: 'skip-agent'}"
+    run: |
+      echo "Installing DOCA: ${DOCA_VERSION} ..."
+      if [ `id -u` -ne 0 ]; then 
+        export SUDO="sudo"
+      else
+        export SUDO=""
+      fi
+      $SUDO .ci/scripts/doca_install.sh 
+  - name: Install Doca-host on Tools
```

Comment:
I am not sure we even need OFED/Doca for style check, we dont compile anything

Reply:
Ack